Remotion/Web/Core/UI/Controls/WebTreeNodeCollection.cs

```diff
@@ -108,5 +110,17 @@ public new WebTreeNode Find (string id)
     //        node.CollapseAll();
     //    }
     //  }
+
+    public static IReadOnlyCollection<WebTreeNode> GroupByCategory (WebTreeNode[] nodes)
+    {
+      var categories = nodes.Select (node => node.Category).Distinct();
```

Comment:
please recheck the comment:
https://github.com/re-motion/Framework/pull/214/files/9127ba937fe0814e848f9b2005807c2ec1ee9827#r717284902